README.md

```diff
@@ -132,23 +133,15 @@ This value can be attained by pointing a browser to the OSIsoft PIWEBAPI URL (ht
 (If the PIWEBAPI certificates are self-signed a security warning may occur when trying to access this page. This warning should be ignored.)
 
 #### Proxy URL
-Adding the field "ProxyURL" under the server config section will change request endpoints to https://domain-name/proxy-path. This field is optional. 
-Below is an example config file with the optional proxy URL in use.
+This optional field is only needed when using a non-default endpoint of https://HOSTNAME/piwebapi. Adding the field "ProxyURL" under the server config section will change request endpoints to https://HOSTNAME/proxy-path. HOSTNAME will be retrieved from the URL field in the configuration file. 
+
+Below is a partial config file with the optional proxy URL in use. Its endpoint is set to https://192.168.1.1/long/proxy/example/omf.
    ```json
    {
    "ServerConfig":{
       "URL":"192.168.1.1",
-      "WebID":"F2DS4knnwtuIDUC1RLy6XJGV4QREVTS1RPdaswQ1MyUDFT",
-      "Credentials":"cGFzc3dvcmQ=",
       "ProxyURL":"long/proxy/example/omf"
-   },
-   "eWONConfig":{
-      "CertificatePath":"/usr"
-   },
-   "AppConfig": {
-       "LoggingLevel": 4,
-       "CommunicationType": "omf"
-       }
+      },
```

Comment:
trailing comma should be removed. Also indentation is improper on the closing bracket